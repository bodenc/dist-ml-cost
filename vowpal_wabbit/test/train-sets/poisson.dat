12 | x:8 y:3
11 | x:7 y:9
3 | x:2 y:1
13 | x:3 y:8
10 | x:7 y:2
5 | x:3 y:2
8 | x:0 y:8
14 | x:7 y:5
14 | x:5 y:8
10 | x:1 y:7
11 | x:4 y:6
11 | x:7 y:4
4 | x:0 y:3
13 | x:9 y:6
14 | x:5 y:9
9 | x:2 y:3
9 | x:4 y:5
4 | x:2 y:2
4 | x:2 y:2
9 | x:5 y:6
2 | x:2 y:0
11 | x:9 y:1
0 | x:0 y:0
9 | x:8 y:9
8 | x:3 y:6
4 | x:1 y:5
8 | x:4 y:4
4 | x:1 y:3
5 | x:4 y:2
15 | x:7 y:8
12 | x:4 y:9
6 | x:9 y:1
14 | x:7 y:4
1 | x:0 y:1
13 | x:8 y:5
9 | x:8 y:4
7 | x:0 y:9
8 | x:4 y:4
7 | x:1 y:7
6 | x:0 y:4
20 | x:9 y:9
6 | x:3 y:1
3 | x:6 y:1
15 | x:9 y:8
6 | x:6 y:1
10 | x:5 y:2
6 | x:0 y:9
14 | x:7 y:7
5 | x:2 y:0
4 | x:1 y:1
24 | x:7 y:8
11 | x:5 y:7
9 | x:6 y:8
3 | x:4 y:2
1 | x:0 y:2
23 | x:7 y:9
19 | x:9 y:9
3 | x:4 y:2
4 | x:1 y:3
11 | x:3 y:5
6 | x:7 y:3
17 | x:2 y:9
15 | x:9 y:8
6 | x:4 y:3
17 | x:8 y:6
9 | x:3 y:2
9 | x:8 y:1
6 | x:4 y:2
15 | x:9 y:8
18 | x:8 y:9
12 | x:6 y:6
11 | x:3 y:7
8 | x:8 y:0
5 | x:3 y:0
7 | x:3 y:4
3 | x:0 y:1
11 | x:3 y:8
9 | x:6 y:5
15 | x:0 y:9
11 | x:8 y:5
8 | x:3 y:7
6 | x:5 y:0
14 | x:6 y:9
0 | x:0 y:0
24 | x:7 y:7
17 | x:5 y:9
18 | x:3 y:9
9 | x:4 y:6
14 | x:8 y:8
8 | x:0 y:8
5 | x:0 y:8
3 | x:3 y:4
8 | x:4 y:9
9 | x:0 y:7
8 | x:0 y:9
0 | x:2 y:2
1 | x:1 y:1
7 | x:7 y:0
4 | x:5 y:0
9 | x:4 y:4