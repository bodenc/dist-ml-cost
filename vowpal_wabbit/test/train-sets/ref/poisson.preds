7.967519
14.577387
3.787162
9.225329
6.519360
4.628412
7.135134
9.204070
10.948843
6.928997
7.985923
8.204552
4.015848
12.254377
12.282683
4.766107
7.118692
4.248531
4.248531
8.699983
3.375894
6.897099
2.844477
15.880821
7.330471
5.505801
6.345636
4.374924
5.042261
12.994356
11.274570
6.897099
8.204552
3.191004
10.027047
8.938160
8.004369
6.345636
6.928997
4.505078
17.300802
4.125789
5.334414
15.422018
5.334414
5.493113
8.004369
11.583235
3.375894
3.476327
12.994356
9.759853
11.927832
5.042261
3.579747
14.577387
17.300802
5.042261
4.374924
6.534418
7.313579
9.499780
15.422018
5.656532
11.248590
4.628412
6.331013
5.042261
15.422018
15.880821
9.477887
8.223503
5.643497
3.677749
5.824812
3.191004
9.225329
8.448636
8.004369
10.027047
8.223503
4.364842
13.380937
2.844477
11.583235
12.282683
10.349201
7.985923
14.156240
7.135134
7.135134
5.824812
11.274570
6.360293
8.004369
4.248531
3.476327
5.180302
4.364842
6.345636
