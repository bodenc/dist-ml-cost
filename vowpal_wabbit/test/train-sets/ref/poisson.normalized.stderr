only testing
predictions = poisson.normalized.preds
Num weight bits = 2
learning rate = 0.5
initial_t = 0
power_t = 0.5
using no cache
Reading datafile = train-sets/poisson.dat
num sources = 1
average  since         example        example  current  current  current
loss     last          counter         weight    label  predict features
95.590408 95.590408            1            1.0  12.0000   9.2347        3
82.029816 68.469223            2            2.0  11.0000  15.2623        3
70.361785 58.693754            4            4.0  13.0000   9.6882        3
66.002784 61.643783            8            8.0  14.0000  10.2895        3
73.706988 81.411191           16           16.0   9.0000   5.4146        3
54.857522 36.008057           32           32.0   6.0000   8.2880        3
71.853323 88.849125           64           64.0   6.0000   6.4692        3

finished run
number of examples per pass = 100
passes used = 1
weighted example sum = 100.000000
weighted label sum = 916.000000
average loss = 74.371941
best constant = 9.160000
total feature number = 276
